client/jetpack-cloud/sections/overview/quick-links-nav/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { category, key, plugins, plus, receipt, store, tag } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { JETPACK_DASHBOARD_QUICK_LINKS_NAV_PREFERENCE } from 'calypso/state/jetpack-agency-dashboard/selectors';
+import {
+	JETPACK_MANAGE_BILLING_LINK,
+	JETPACK_MANAGE_DASHBOARD_LINK,
+	JETPACK_MANAGE_INVOICES_LINK,
+	JETPACK_MANAGE_LICENCES_LINK,
+	JETPACK_MANAGE_PLUGINS_LINK,
+} from '../../sidebar-navigation/lib/constants';
+import FoldableNav from '../foldable-nav';
+import { FoldableNavItem } from '../foldable-nav/types';
+
+export default function QuickLinksNav() {
+	// JETPACK_DASHBOARD_QUICK_LINKS_NAV_PREFERENCE
+
+	const translate = useTranslate();
+
+	const header = translate( 'Quick links' );
+	const tracksName = 'calypso_jetpack_manage_overview_quick_links';
+
+	const navItems: FoldableNavItem[] = [
+		{
+			icon: category,
+			link: JETPACK_MANAGE_DASHBOARD_LINK,
+			title: translate( 'Manage all sites' ),
+		},
+		{
+			icon: plus,
+			link: localizeUrl( 'https://wordpress.com/jetpack/connect' ),
+			title: translate( 'Add sites to Jetpack Manage' ),
```

Comment:
Maybe we should shorten this so it doesn't wrap on smaller screens...

Reply:
Or we can fix the minimum size of the sidebar... it's something that is not specified...

Reply:
For now I'll leave it, but we can review it holistically when everything else is in place.